lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/sys.rb

```diff
@@ -395,7 +395,16 @@ def cmd_shell(*args)
   #
   def pty_shell(sh_path, raw: false)
     args = ['-p']
-    args << '-r' if raw
+
+    if raw
+      args << '-r' if raw
```

Comment:
Uhhhhhhh what? I think this is left over from previous iteration of code, no need to check this twice.

```suggestion
    if raw
      args << '-r'
```